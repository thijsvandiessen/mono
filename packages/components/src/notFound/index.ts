export { NotFound } from './notFound.jsx'
