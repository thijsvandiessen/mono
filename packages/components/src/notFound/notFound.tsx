import React from 'react'
import { Link } from '@mono/next-js'

export const NotFound = () => {
  return (
    <section className="content-layout">
      <h2>Niet gevonden</h2>
      <p>Deze pagina is niet gevonden.</p>
      <p>
        <Link href="/">Ga terug naar de homepage</Link>
      </p>
    </section>
  )
}
