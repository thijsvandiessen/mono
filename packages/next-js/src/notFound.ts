export { notFound } from 'next/navigation.js'
