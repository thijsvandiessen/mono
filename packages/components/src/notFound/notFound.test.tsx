import React from 'react'
import { render } from '@testing-library/react'
import { NotFound } from './notFound'

describe('NotFound component', () => {
  it('shows all the data', () => {
    const { container } = render(<NotFound />)
    expect(container).toMatchSnapshot()
  })
})
