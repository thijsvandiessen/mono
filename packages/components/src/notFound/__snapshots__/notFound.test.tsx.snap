// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component shows all the data 1`] = `
<div>
  <section
    class="content-layout"
  >
    <h2>
      Niet gevonden
    </h2>
    <p>
      Deze pagina is niet gevonden.
    </p>
    <p>
      <a
        href="/"
      >
        Ga terug naar de homepage
      </a>
    </p>
  </section>
</div>
`;
